src/scripts/data_file_processing/healthsystem/equipment/equipment_availability_estimation.py

```diff
@@ -322,6 +322,12 @@
                                                   .drop_duplicates() \
                                                   .pipe(lambda x: x.set_index(x['Item_code'].astype(int)))['Category'] \
                                                   .to_dict()
+# Manually declare the price category for equipment items added manually
+# 402: Endoscope: 'Cost >= $1000'
+equipment_price_category_mapper[402] = 'Cost >= $1000'
+# 403: Electrocardiogram: 'Cost >= $1000'
+equipment_price_category_mapper[403] = 'Cost >= $1000'
+
```

Comment:
@sakshimohan  -- I wasn't sure if this was the best place to encode the cost of these additional pieces of equipment that Joe/Eva introduced, or the file on dropbox. It's fine with me to leave here it here, if it's Ok with you.